test/e2e/deploy/deploy.go

```diff
@@ -67,6 +67,14 @@ func WaitForOperator(c *Cluster, namespace string) *v1.Pod {
 	return operator
 }
 
+// WaitForOLM watis till the "olm-operator" Pod in given namspace
+// is ready else fails with exception.
+func WaitForOLM(c *Cluster, namespace string) *v1.Pod {
+	olm := c.WaitForAppInstance("olm-operator", "", namespace)
+	ginkgo.By("OLM is ready!")
```

Comment:
It's better to put the `ginkgo.By` *before* something that can block a while, because then it is obvious from a log what is currently going it.